source/app/blueprints/case/case_timeline_routes.py

```diff
@@ -627,6 +627,7 @@ def case_edit_event(cur_id, caseid):
         event.case_id = caseid
         event.event_added = datetime.utcnow()
         event.user_id = current_user.id
+        event.event_tags = request_data.get(u'event_tags')
```

Comment:
Oh I missed that... that was a mistake because I thought that was missing I didnt quite understand how the event_tags where handled. This can be removed.

Reply:
_Hint for us_: to remove